run/storage-streaming/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "storage-streaming",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=60000 --exit",
+    "run-chunked-upload": "functions-framework --target=chunkedUpload --source chunked-http.js",
+    "run-chunked-download": "functions-framework --target=chunkedDownload --source chunked-http.js",
+    "run-streaming-copy": "functions-framework --target=streamingCall --source copying-files.js",
+    "run-non-streaming-copy": "functions-framework --target=nonStreamingCall --source copying-files.js"
+  },
+  "dependencies": {
+    "@google-cloud/storage": "^5.8.5",
+    "busboy": "^0.3.1",
+    "mocha": "^8.0.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "child-process-promise": "^2.2.1",
+    "request": "^2.88.2",
```

Comment:
We need to move away from `request` due to it's deprecation.  Mind following the pattern we laid out with gaxios and wait-port in other PRs?

Reply:
This sample is still being worked on - but yes, I can make sure it doesn't use `request`.